### PR TITLE
Fixed JSON unmarshaling error replacing Campaign object by the docume…

### PR DESCRIPTION
**Pull request by dgrr**
…nted in the API. https://mediamath.api-docs.io/v2/campaigns/get-campaigns

Fixes https://github.com/MediaMath/go-t1/issues/9

### CODE REVIEW

**Comment by pswaminathan** [comment]
I'm no longer a maintainer here, so I'm just in the peanut gallery. But this changes field names in ways that need to be addressed. e.g. `IOName` → `IoName`, `DCSDataIsCampaignLevel` → `DcsDataIsCampaignLevel`. Not only do I think it's not great to slide in name changes like this, but I think the original styling should be preferred. The reason I made them the way they are is that the Go conventions directly address this case in [their common code review comments](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

**Comment by dgrr** [comment]
Ok. I am going to change those names. I used [json-to-go](https://mholt.github.io/json-to-go/) to parse json data and I just copied the structure from there.

File: models/campaign.go [hunk 1]
```diff
@@ -18,47 +18,73 @@ import (
...
+type GoalValue struct {
+	CurrencyCode string `json:"currency_code"`
+	Value        int    `json:"value"`
+}
+
+type SpendCapAmount struct {
+	CurrencyCode string `json:"currency_code"`
+	Value        int    `json:"value"`
+}
+
```
Comment (FodT) [inline models/campaign.go]: Is there a reason you've created three essentially identical structs rather than a single `Currency` struct?

Comment (dgrr) [reply models/campaign.go]: No. It's just to help programmer to identify the types with different names. I can create a common type and name the other three.

Comment (FodT) [reply models/campaign.go]: please do, this is a common type in our api, and they're not going to diverge.

**Pull request closed without merging**